client/src/components/squids/hooks/useSquidList.ts

```diff
@@ -0,0 +1,37 @@
+import { useQuery, UseQueryResult } from "react-query";
+
+import { ApiClient } from "../../../backend/ApiClient";
+
+export const useSquidList = ({
+  pageIndex,
+  itemsPerPage,
+}: {
+  pageIndex: number;
+  itemsPerPage: number;
+}): UseQueryResult<SquidData> =>
+  useQuery(
+    ["squids", { pageIndex, itemsPerPage }],
+    () =>
+      ApiClient.get("/squids", {
+        params: {
+          limit: itemsPerPage,
+          offset: pageIndex * itemsPerPage,
+        },
+      }).then((resp) => resp.data),
+    {
+      staleTime: Infinity,
+    }
+  );
+
+interface SquidData {
+  pageCount: number;
+  squids: Squid[];
+}
+
+export interface Squid {
```

Comment:
Now that we've got this in place, I would suggest moving this interface into its own file, since you're going to (and already have) ended up using it in lots of places. We're still refining our pattern, but I suggest`./types/SquidShape.ts` and within that, 

```
export interface SquidShape {
...
}
```

We name it `SquidShape` instead of `Squid` because we're defining what the json for a squid should look like, rather than defining the attributes of an instance of a Squid model